components/eamxx/data/scream_default_output.yaml

```diff
@@ -63,15 +63,7 @@ output_control:
 # WARNING: ERS/ERP tets will override this with STOP_N/STOP_OPTION
   Frequency: ${HIST_N}
   frequency_units: ${HIST_OPTION}
-  MPI Ranks in Filename: false
```

Comment:
I think we should keep this line in, it alerts new users to the option exisiting even if it is default false.

Reply:
Well, I don't really want them to use it... That option is really just for unit tests...

Reply:
That's fair